homeassistant/components/rfxtrx.py

```diff
@@ -434,46 +345,39 @@ def update_state(self, state, brightness=0):
         """Update det state of the device."""
         self._state = state
         self._brightness = brightness
-        if self.added_to_hass:
```

Comment:
I think this will reintroduce this bug: https://github.com/home-assistant/home-assistant/issues/9751

Reply:
No, with my cleanup it will be fix correct now. At the moment all things are very wired

Reply:
OK, great.
Do you know when you will finish this?
This issue https://github.com/home-assistant/home-assistant/issues/10464 is a problem for many users and https://github.com/home-assistant/home-assistant/pull/10203 would have fixed it